fetch.bs

```diff
@@ -3824,6 +3824,45 @@ the request.
  <var>request</var>'s <a for=request>origin</a> to  <var>request</var>'s <a for=request>client</a>'s
  <a for="environment settings object">origin</a>.
 
+ <li>
+  <p>If all of the following conditions are true:</p>
+
+  <ul class=brief>
+   <li><p><var>request</var>'s <a for=request>URL</a>'s <a for=url>scheme</a> is an
+   <a>HTTP(S) scheme</a>
+
+   <li><p><var>request</var>'s <a for=request>mode</a> is "<code>same-origin</code>",
+   "<code>cors</code>", or "<code>no-cors</code>"
+
+   <li><p><var>request</var>'s <a for=request>window</a> is not null
+
+   <li><p><var>request</var>'s <a for=request>method</a> is `<code>GET</code>`
+
+   <li><p><var>request</var>'s <a for=request>unsafe-request flag</a> is not set, or
+   <var>request</var>'s <a for=request>header list</a> <a for="list">is empty</a>
+  </ul>
+
+  <p>then perform the following steps:
+
+  <ol>
+   <li><p>Assert: <var>request</var>'s <a for=request>origin</a> is <var>request</var>'s
+   <a for=request>client</a>'s <a for="environment settings object">origin</a>.
+
+   <li><p>Let <var>onPreloadedResponseAvailable</var> be to an algorithm that runs the following
+   step <a>in parallel</a> given <a for=/>response</a> <var>response</var>: run
+   <a>fetch finale</a> given <var>response</var> and <var>fetchParams</var>.
+
+   <li><p>Let <var>foundPreloadedResource</var> be the result of invoking
+   <a>consume a preloaded resource</a> for <var>req</var>'s <a for=request>window</a>,
```

Comment:
Is "consume a preloaded resource" defined elsewhere? I'm failing to find it..

Reply:
https://html.spec.whatwg.org/#consume-a-preloaded-resource